README.md

```diff
@@ -45,13 +45,13 @@ Reason for each dependency:
 
 For resource constrained platforms we will split te build into 2 steps to make sure not to overflow the memory
```

Comment:
I know you didn't add this, but while you are modifying it, would you mind fixing "te" -> "the"?  Thanks.

Reply:
@clalancette if the PR allows to apply changes (which this one seems to) you can do this simple change directly through the web interface and don't have to wait for the author of the patch to apply it.